CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 Changes to Calva.
 
 ## [Unreleased]
+- [Use cljs for state](https://github.com/BetterThanTomorrow/calva/pull/1053)
```

Comment:
I don't think it belongs in the changelog really, since to the user things should stay the same. But it gets relevant if something break. Then our users (since they are programmers) could connect the dots and help us find issues.

All in all, I think it should stay, but be marked as being of internal tech nature.
```suggestion
- Implementation detail: [Use cljs for state](https://github.com/BetterThanTomorrow/calva/pull/1053)
```